superset/sql/parse.py

```diff
@@ -654,6 +654,16 @@ def is_mutating(self) -> bool:
         if isinstance(self._parsed, exp.Command) and self._parsed.name == "ALTER":
             return True  # pragma: no cover
 
+        if (
```

Comment:
should this be a super simple method in `db_engine_spec` that's a no-op in base and something different for PG?